README.md

```diff
@@ -18,7 +18,7 @@ You might also be interested in becoming a [tester](https://forum.k9mail.app/t/h
 ## Release Notes
 
 Check out the [Release Notes](https://github.com/thunderbird/thunderbird-android/wiki/ReleaseNotes) to find out what changed
-in each version of K-9 Mail.
+in each version of K-9 Mail. For a more detailed list of changes, see the XML [Changelog](app/ui/legacy/src/main/res/raw/changelog_master.xml).
```

Comment:
The XML file of the last beta/stable release is linked from the ReleaseNotes wiki page. So it's probably not entirely accurate to say it's a more detailed list of changes. Maybe more "up to date"?